crates/net/network/src/manager.rs

```diff
@@ -676,15 +676,15 @@ where
                             let total_active =
                                 this.num_active_peers.fetch_add(1, Ordering::Relaxed) + 1;
                             this.metrics.connected_peers.set(total_active as f64);
-                            info!(
-                                target : "net",
+                            debug!(
+                                target: "net",
                                 ?remote_addr,
                                 %client_version,
                                 ?peer_id,
                                 ?total_active,
+                                kind=%direction,
                                 "Session established"
                             );
-                            debug!(target: "net", kind=%direction, peer_enode=%NodeRecord::new(remote_addr, peer_id), "Established peer enode");
```

Comment:
thoughts on adding the enode back? it was useful when investigating suspicious peers